src/System.Runtime.Intrinsics.Experimental/ref/System.Runtime.Intrinsics.cs

```diff
@@ -25,6 +25,148 @@ public struct Vector256<T> where T : struct
          private int _dummy;
     }
 }
+namespace System.Runtime.Intrinsics.Arm.Arm64
+{
+    public static class Simd
+    {
+        public static bool IsSupported { get { throw null; } }
+        public static Vector64<byte>    Abs(Vector64<sbyte>   value) { throw null; }
+        public static Vector64<ushort>  Abs(Vector64<short>   value) { throw null; }
```

Comment:
The API follows pattern in Abs that converts signed integrals to unsigned ones. IMO opinion it would be better to keep original signed type as return vector type since it is very probable that users will continue using signed integrals of the same type as argument type.

Furthermore, Abs return values will always fir in signed integrals of the same type.

Reply:
This is what is implemented in CoreCLR.  I am happy to change, but I will wait for consensus first.

I can pull these from the existing PR so that the rest of these can be merged.

Reply:
This is entirely orthogonal to this PR, as this is what is being done for the x86 intrinsics. I think we should merge this PR as-is, and then consider whether or not to change it for all targets.

Reply:
@4creators, @sdmaclea. The reason x86 does it this way is because the instructions explicitly document themselves as: `Compute the absolute value of bytes in xmm2/m128 and store UNSIGNED result in xmm1.`